azext_iot/iothub/params.py

```diff
@@ -45,3 +50,189 @@ def load_iothub_arguments(self, _):
             help="Maximum interval of time, in seconds, that the digital twin command will wait for the result.",
             arg_group="Timeout"
         )
+
+    with self.argument_context("iot device") as context:
+        context.argument(
+            "auth_type_dataplane",
+            options_list=["--auth-type"],
+            arg_type=hub_auth_type_dataplane_param_type,
+        )
+        context.argument("data", options_list=["--data", "--da"], help="Message body.")
+        context.argument(
+            "properties",
+            options_list=["--properties", "--props", "-p"],
+            help=info_param_properties_device(),
+        )
+        context.argument(
+            "msg_count",
+            options_list=["--msg-count", "--mc"],
+            type=int,
+            help="Number of device messages to send to IoT Hub.",
+        )
+        context.argument(
+            "msg_interval",
+            options_list=["--msg-interval", "--mi"],
+            type=int,
+            help="Delay in seconds between device-to-cloud messages.",
+        )
+        context.argument(
+            "receive_settle",
+            options_list=["--receive-settle", "--rs"],
+            arg_type=get_enum_type(SettleType),
+            help="Indicates how to settle received cloud-to-device messages. "
+            "Supported with HTTP only.",
+        )
+        context.argument(
+            "protocol_type",
+            options_list=["--protocol", "--proto"],
+            arg_type=get_enum_type(ProtocolType),
+            help="Indicates device-to-cloud message protocol",
+        )
+        context.argument(
+            "device_symmetric_key",
+            options_list=["--symmetric-key", "--key"],
+            arg_group="Device Authentication",
+            help="Symmetric key to use for the device.",
```

Comment:
We should mention here that Symmetric key auth takes precedence over x509 auth using certificate and key files (with passphrase)